.github/workflows/deploy_preview.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy
+on:
+  push:
+    branches-ignore:
+      - main
+    paths:
+      - 'src/**'
+      - 'docs/**'
+      - '.github/workflows/deploy*.yml'
+      - 'package.json'
+jobs:
+  deploy-preview:
+    if: ${{ github.repository == 'primer/css' }}
+    name: Preview
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: chrnorm/deployment-action@releases/v1
+        name: Create GitHub deployment
+        id: deployment
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          environment: Preview
+
+      - name: Vercel Action
+        uses: amondnet/vercel-action@v20.0.0
+        id: vercel-action
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          vercel-token: ${{ secrets.VERCEL_TOKEN_SHARED }}
+          github-comment: false
```

Comment:
Is there a reason we don't want vercel to make a github comment?

Reply:
I thought it was extra noise we didn't really need since there's deployment statuses in the PR also. but that's my personal preference, I'm open to others thoughts. We also have the option to put custom content in here.